pkg/scheduler/framework/runtime/framework.go

```diff
@@ -693,14 +736,28 @@ func (f *frameworkImpl) RunPreFilterExtensionRemovePod(
 	podInfoToRemove *framework.PodInfo,
 	nodeInfo *framework.NodeInfo,
 ) (status *framework.Status) {
+	logger := klog.FromContext(ctx)
+	logger = klog.LoggerWithName(logger, "RemovePod")
+	// We are adding LoggerWithValues for contextualized logging here
+	// but contextualized logging is feature gated and optional until
+	// it hits GA when it will be defaulted. For that reason
+	// logger.V(<LOGLEVEL>).Info or Error calls still need to include
+	// WithValues Key/Values while contextualized logging does not hit GA
+	// (duplicates get filtered out if contextualized logging is enabled)
+	// TODO(knelasevero): Remove duplicated keys from log entry calls
+	// When contextualized logging hits GA
+	// https://github.com/kubernetes/kubernetes/issues/111672
+	logger = klog.LoggerWithValues(logger, klog.KObj(podToSchedule), "node", klog.KObj(nodeInfo.Node()), "extensibleAPI", "PreFilter")
```

Comment:
Missed a key `pod` here.

Reply:
Previous PR https://github.com/kubernetes/kubernetes/pull/111155 missed it. I have added the pod info back.